src/window.rs

```diff
@@ -93,11 +94,23 @@ fn main() {
     ).exit_on_esc(true)
         .build()
         .unwrap();
+    
+    // Create font for displaying text.
```

Comment:
I have two problems with this font creating:
1. It happens at runtime, and if you have moved the binary elsewhere, the font isn't in this folder (I think).
2. It's unnecessary to display the generation.

Reply:
I agree with your first problem but disagree with the second, I think it **can** be handy to see how many generations have passed. It could also be very unnecessary of course but I would like to see it as an option.